src/routes/feedback/components/FeedbackHeader.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import * as React from 'react'
+import styled from 'styled-components'
+import CleanLink from '../../../modules/common/components/CleanLink'
+import { translate } from 'react-i18next'
+import type { TFunction } from 'react-i18next'
+import Dropdown from 'react-dropdown'
+import { Description } from './FeedbackBox'
+import type { LocationState } from 'redux-first-router'
+import { goToFeedback } from '../../../modules/app/routes/feedback'
+import FeedbackDropdownItem from '../FeedbackDropdownItem'
+
+const Header = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+`
+
+const CloseButton = styled(CleanLink)`
+  font-size: 2rem;
+`
+
+const Title = styled.div`
+  padding: 15px 0 10px;
+  font-size: ${props => props.theme.fonts.subTitleFontSize};
+`
+
+type PropsType = {
+  location: LocationState,
+  feedbackOptions: Array<FeedbackDropdownItem>,
+  selectedFeedbackOption: FeedbackDropdownItem,
+  onFeedbackOptionChanged: FeedbackDropdownItem => void,
+  t: TFunction
+}
+
+export class FeedbackHeader extends React.Component<PropsType> {
+  render () {
+    const {t, location, selectedFeedbackOption, feedbackOptions, onFeedbackOptionChanged} = this.props
+
+    return (
+      <React.Fragment>
+        <Header>
+          <Title>{t('feedback')}</Title>
+          <CloseButton to={goToFeedback(location)}>x</CloseButton>
```

Comment:
Also we may want to consider using `<CloseButton to={goToFeedback(location)} replace={true}>`. That way, after clicking on the CloseButton and then clicking back, doesn't open the feedback-menu again, but rather navigate to the page before that... I think that's what users would expect (at least me^^)

Reply:
That way, clicking on the back button does nothing, because the stack is like this:
Page1 < Page2 < FeedbackForPage2. So if you click on the close button/overlay/submit button with replace=true, you remove FeedbackForPage2 and add Page2 to the history, so it looks like this: Page1 < Page2 < Page2, so a click on back does nothing. I would leave it like this for now.

Reply:
Oh yeah, you're right. Convinced :)